### ISSUE TITLE
GoalKeeper animation out of sync.

### ISSUE THREAD

**Issue by NextStarIndustries** [PapaJoesSoup/BDArmory#495]
The Goalkeeper barrel rotation animation doesn't activate while firing, it activates after its done firing.

**Comment by SpannerMonkey** [comment]
Unable to replicate this problem using BDA 1.1.0.38488_03172018
Or any of the current 1.4x development versions

Of note in the case of the goalkeeper as part of the fire animation it has a spin down time, so the barrel continues to rotate for a short time after the firing has ceased, this is not a bug but the intended operation.

**Comment by NextStarIndustries** [comment]
The barrel doesn't rotate at all until after you release the fire button. It's extremely easy to replicate just use the thing in any version and press the fire button. If this is how you want to be then so be it, but in my opinion it looks wonky and it should rotate while firing like the others do.

**Comment by XOC2008** [comment]
Goalkeeper barrels rotate just fine on my end.

**Comment by NextStarIndustries** [comment]
I'll just post a video for everyone to check out.

**Comment by SpannerMonkey** [comment]
Videos are useless in diagnosing or confirming the issue Logs are required. All a video does is confirm that your particular installation has an issue.

**Comment by DoctorDavinci** [comment]
No problems with the barrels here ... tested BDAc v1.1 Final and KSP v1.4.2

There has been issues in the past with other mods interfering with BDAc operation such as NSI BDArmory Weapons Extension ... Please check your install and try to replicate the issue again in a clean game with only BDAc and it's dependencies

If the issue persits then we will need to see logs

**Comment by SpannerMonkey** [comment]
Hi, as we have tried to replicate your issue and been unable, a video as mentioned just proves it doesnt work for you,  it does not demonstrate the why of your problem.  Without a log we don't know anything about your game environment and or how it affects BDA.
I resent the implication that we have ignored or ridiculed your bug report , as i for one tested in 4 versions of BDA in 3 versions of KSP  and was in each case unable to replicate your issue . I have acted in a proper manner throughout and treated your report as genuine.
